citycouncil/settings.py

```diff
@@ -185,10 +188,3 @@ def __mod__(self, other):
 # by commenting out EMAIL_BACKEND and copying all the production email settings.
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 GOOGLE_EMBED_API_KEY = os.getenv('GOOGLE_EMBED_API_KEY')
-
-django_heroku.settings(
-    locals(),
-    databases=not os.environ.get('CI'),
-    staticfiles=not os.environ.get('CI'),
-    test_runner=False,  # doesn't work with sqlite3
-)
```

Comment:
Removed this to the prod_settings file because it looks like that may include CI https://github.com/seakelps/cambridge/pull/190/files#diff-b72968508bf490c844b98c697207d16cebad4bfa4191e69e470d5c6cd60be9e7R10-R11